### DISCUSSION TITLE
Disabling the hausdorf measurement

### DISCUSSION THREAD

**Discussion by ataML** [FETS-AI/Challenge#98] [Q&A] [answered]
Hi,

how can I disable the hausdorf measurement?

**Comment by sarthakpati** [comment, marked as answer]
You can try passing the kwarg called `include_validation_with_hausdorff` with the value `False` when calling `run_challenge_experiment` within your notebook [[ref](https://github.com/FETS-AI/Challenge/blob/main/Task_1/fets_challenge/experiment.py#L238)].

Please note that this flag will be reset during the testing phase, thereby always enabling the calculation of Hausdorff.

**Reply by ataML** [reply]
Thanks a lot for the quick response
